common/persistence/elasticsearch/decodeBench_test.go

```diff
@@ -43,6 +43,7 @@ var (
          "WorkflowID": "6bfbc1e5-6ce4-4e22-bbfb-e0faa9a7a604-1-2256",
          "WorkflowType": "TestWorkflowExecute",
  		 "Encoding" : "thriftrw",
```

Comment:
nit: indent?

Reply:
I do not see this in my editor for some reason.